File: pkg/codegen/testing/test/testdata/simple-splat-pp/nodejs/simple-splat.ts
import * as pulumi from "@pulumi/pulumi";
import * as splat from "@pulumi/splat";

const allKeys = splat.getSshKeys({});
const main = new splat.Server("main", {sshKeys: allKeys.then(allKeys => allKeys.sshKeys.map(__item => __item.name))});

File: pkg/codegen/testing/test/testdata/simple-splat-pp/dotnet/simple-splat.cs
using System.Collections.Generic;
using System.Linq;
using Pulumi;
using Splat = Pulumi.Splat;

return await Deployment.RunAsync(() => 
{
    var allKeys = Splat.GetSshKeys.Invoke();

    var main = new Splat.Server("main", new()
    {
        SshKeys = allKeys.Apply(getSshKeysResult => getSshKeysResult.SshKeys).Select(__item => __item.Name).ToList(),
    });

});

